src/components/routes/Member/AuthenticatedMember/UpdatePasswordRoute/UpdatePassword.jsx

```diff
@@ -54,7 +54,9 @@ class UpdatePassword extends React.Component {
   };
 
   renderButton = () => (
-    <Button className="login-button button__login" type="submit" value="Update" />
+    <div className={'update-credentials__button-container'}>
```

Comment:
Why we are not using classNames as per the convention. It should be
```suggestion
    <div className={'update-password__button-container'}>
```

Reply:
DONE.